gfx_scene.h

```diff
@@ -244,22 +245,22 @@ GfxMetadata const &gfxSceneGetImageMetadata(GfxScene scene, uint64_t image_handl
 //! Image helpers.
 //!
 
-inline DXGI_FORMAT gfxImageGetFormat(GfxImage const &image)
```

Comment:
Made ```gfxImageGetFormat``` private as there is already a member variable of ```GfxImage``` that should be probed for the internal format (as with compressed formats the possible formats gets significantly larger than what this function could correctly handle.

Reply:
Agreed.

Reply:
Well there is now the ```gfxImageIsFormatCompressed``` function that is added in place of the removed ```gfxImageGetFormat``` which still classifies as an image helper???

Reply:
Oh right I see, then it's certainly good!
Wasn't visible from the small github previous 😅

Reply:
Yeah it shows up down below somewhere :smile: